recipes/mapd-core-cpu/build.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+set -ex
+
+env
+
+ls -la $BUILD_PREFIX/bin
+
+# sed -i option is handled differently in Linux and OSX
+if [ $(uname) == Darwin ]; then
+    INPLACE_SED="sed -i \"\" -e"
+else
+    INPLACE_SED="sed -i"
+fi
+
+# mapd-core v 4.5.0 (or older) hardcodes /usr/bin/java. Grab
+# Calcite.cpp with a fix
+# (https://github.com/omnisci/mapd-core/pull/316) from a repo:
+wget https://raw.githubusercontent.com/omnisci/mapd-core/7c1faa09dd88d0cc735b629048f74d71baa9179f/Calcite/Calcite.cpp
+mv Calcite.cpp Calcite/
+
+# conda build cannot find boost libraries from
+# ThirdParty/lib. Actually, moving environment boost libraries to
+# ThirdParty/lib does not make much sense. The following is just a
+# quick workaround of the problem. Upstream will remove the relevant
+# code from CMakeLists.txt as not needed.
+$INPLACE_SED 's/DESTINATION ThirdParty\/lib/DESTINATION lib/g' CMakeLists.txt
```

Comment:
You can use a different delimiter in sed, if you are using slashes. E.g. colons,

`sed -i  's:DESTINATION ThirdParty/lib:DESTINATION lib:g' CMakeLists.txt `